src/reactor.h

```diff
@@ -0,0 +1,242 @@
+#ifndef CYCLUS_TRICYCLE_REACTOR_H_
+#define CYCLUS_TRICYCLE_REACTOR_H_
+
+#include <string>
+
+#include "cyclus.h"
+#include "boost/shared_ptr.hpp"
+
+namespace tricycle {
+
+/// @class Reactor
+///
+/// This Facility is intended
+/// as a skeleton to guide the implementation of new Facility
+/// agents.
+/// The Reactor class inherits from the Facility class and is
+/// dynamically loaded by the Agent class when requested.
+///
+/// @section intro Introduction
+/// Place an introduction to the agent here.
+///
+/// @section agentparams Agent Parameters
+/// Place a description of the required input parameters which define the
+/// agent implementation.
+///
+/// @section optionalparams Optional Parameters
+/// Place a description of the optional input parameters to define the
+/// agent implementation.
+///
+/// @section detailed Detailed Behavior
+/// Place a description of the detailed behavior of the agent. Consider
+/// describing the behavior at the tick and tock as well as the behavior
+/// upon sending and receiving materials and messages.
+class Reactor : public cyclus::Facility  {
```

Comment:
You may want to call this a `FusionPowerPlant` just to be clear that it's different from a fission `Reactor`